specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/DscpConfigurationGet.json

```diff
@@ -14,8 +14,12 @@
         "properties": {
           "provisioningState": "Succeeded",
           "associatedNetworkInterfaces": [
```

Comment:
>associatedNetworkInterfaces [](http://example.com/codeflow?start=11&length=27)

This has other fields too as per the swagger. Are those properties not being shown in the response ?

Please update all the relevant fields for all files.